src/screens/profile-screen/AlbumsTab.tsx

```diff
@@ -9,12 +12,17 @@ const messages = {
 
 export const AlbumsTab = () => {
   const { profile, albums } = useSelectorWeb(getProfile)
-  if (!profile || !albums) return null
+
+  const userAlbums = useMemo(() => {
+    if (profile && albums)
```

Comment:
I prefer curlies for single line conditionals, but I don't think we have that in the styleguide

Reply:
yeah agreed, was missed!